### ISSUE TITLE
Purchased Ticket 'Checked in' value not displaying in the list view

### ISSUE THREAD

**Issue by anchovy** [verbb/Events#106]
### Describe the bug

When looking at the Purchased Tickets list view the 'Checked in' value isn't coming through, it's always showing false.

### Steps to reproduce

1. Add an event
2. Purchased a ticket
3. Scan the QR code
4. View the CP _events/purchased-tickets_ 
6. 'Checked in' column will display false

![image](https://user-images.githubusercontent.com/599005/189770099-793f48d3-f768-4850-add9-824d79e690f3.png)

### Craft CMS version

Craft Pro 3.7.53.1

### Plugin version

1.4.22

### Multi-site?

No

### Additional context

The checked in status of the purchased ticket detail page correctly shows the 'true' Checked in status:
![image](https://user-images.githubusercontent.com/599005/189770140-3dd0d90d-d2c6-4ab0-9560-223beb0c8e2e.png)

**Comment by engram-design** [comment]
Strange, I'm not seeing that on my end. 
![image](https://user-images.githubusercontent.com/1221575/189777090-a7329842-4b50-4080-be46-12de7ddc43df.png)

With their respective rows in the database:

![image](https://user-images.githubusercontent.com/1221575/189777104-65918f40-d893-4b94-92ca-f6d1c05a2c32.png)

Appreciate the [PR](https://github.com/verbb/events/pull/108/commits/e7c4d4906010a59cac53ab738eb7552c13094dc0) but I don't think that's quite right, to re-fetch the record. We shouldn't need to do that, as the element's properties should already be set to this content. Something else must be going on...

Everything look okay on your end with the database?

**Comment by anchovy** [comment]
Hi @engram-design That's odd - definitely not working for me locally or on dev (AWS larval forge provisioned server):

![Screenshot 2022-09-13 at 12 27 16 PM](https://user-images.githubusercontent.com/599005/189781505-1ae25628-bc86-4c96-a46e-f39886595b6f.png)

**Comment by engram-design** [comment]
That's very, very strange then. I've even spun up a new site to test, just in case there's something different in my test install, but working correctly for me.

Does the "Checked In Date" column show anything (you might have to add it)?

**Comment by anchovy** [comment]
Hi @engram-design nope, I checked that as well and it's coming through as empty.

**Comment by anchovy** [comment]
Hi @engram-design - any other ideas/debugging I can do to help solve this issue?

**Comment by engram-design** [comment]
I have no idea what's going on here sorry, that doesn't make any sense! What PHP and MySQL version are you on?

Don't suppose it's possible to get your database?

**Comment by anchovy** [comment]
Hi @engram-design - think I've worked out the issue - is there some sort of cache on the purchased tickets line items/view?

Works:
1. purchase a ticket
2. check-in.
3. view purchased tickets
4. checked-in status = true

Doesn't work.
1. purchase a ticket
2. view purchased tickets (checked in status = false)
3. check-in.
4. view purchased tickets
5. checked in status = false

If I then go to utilities and clear all the caches (Seems like it's the `data caches` cache that's needed to be cleared) & revisit the purchased tickets page checked-in status = true.

**Comment by engram-design** [comment]
Good find, this is totally a bug. It looks like it's due to using records (https://github.com/verbb/events/commit/c6d7613aee98aad0f1580fed5a23426074eb909d) and not elements. So you're right in that the element isn't updated but the record (`purchased_tickets`).

Looking at the code, I was wondering why I'd written it like that, but looks like this originated from https://github.com/verbb/events/pull/50

Fixed for the next release. To get the fix early, change your `verbb/events` requirement in `composer.json` to:

```
"require": {
  "verbb/events": "dev-craft-3 as 1.4.22",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.23](https://github.com/verbb/events/releases/tag/1.4.23)
